mobify/documentation-theme
# Only show version picker arrow if JSON loads

Prevents case of a ghostly lonesome down caret icon from being shown if versions JSON doesn't load correctly.

## How to Test
* `npm link`
* `npm link @mobify/documentation-theme` into Web or AMP SDK
* Temporarily delete versions.json from `docs/public/latest/` folder
* `npm run mobify-docs:preview`
* Visit http://localhost:9000 and see that version picker area is simply blank
